main.js

```diff
@@ -0,0 +1,50 @@
+//Exercise 1:
+function isEqual(arr1, arr2) {
+    return arr1.toString() == arr2.toString()
+}
+
+//Exercise 2:

```

Comment:
```suggestion
// Exercise 2:
```
Nhìn kỹ cách đặt dấu cách